src/Microsoft.ML.StandardTrainers/Standard/Online/OnlineLinear.cs

```diff
@@ -61,7 +61,7 @@ public abstract class OnlineLinearOptions : TrainerInputBaseWithLabel
         [BestFriend]
         internal class OnlineDefault
         {
-            public const int NumberOfIterations = 1;
+            public const int NumberOfIterations = 10;
```

Comment:
Modifying within `OnlineLinear.cs` seems to be affecting multiple trainers besides AveragedPerceptron. You may need to find a way to target just AveragedPerceptron.

Reply:
I can do that. Question though, would it be bad to update it for them all? Or is just that we know for AveragedPerceptron 10 is better, but thats not true for the others?

Reply:
For the other trainers, it's unknown if 10 iterations will be better or worse. For most trainers, `NumberOfIterations`/`NumberOfTrees` and [`LearningRate`](https://en.wikipedia.org/wiki/Learning_rate) are the primary hyperparameters; changing them has a large impact.

Since the existing defaults in the other trainers are backed by some level of benchmarking when they were created, I would keep with the existing value. Moving from 1 to 10 iterations, we know the runtime will increase and we should demonstrate the value (accuracy gain) of this extra time spent.

For AveragedPerceptron, we did benchmarks showing the gain in having 10 iterations as the new default. 

Subset of the AveragedPerceptron benchmarks are in the original issue -- https://github.com/dotnet/machinelearning/issues/2305.